proto/substrait/algebra.proto

```diff
@@ -206,9 +206,11 @@ message AggregateRel {
   Rel input = 2;
 
   // A list of expression grouping that the aggregation measured should be calculated for.
+  // Required if there are no measures
   repeated Grouping groupings = 3;
 
   // A list of one or more aggregate expressions along with an optional filter.
```

Comment:
Should this change to zero or more?

Reply:
Per the [Aggregate Properties](https://substrait.io/relations/logical_relations/#aggregate-properties) portion of the spec, both groupings and measures are optional lists of one or more elements.

As written, it appears that the intent is that the empty list should be represented as null. It might be worth loosening this requirement, but that becomes a spec change beyond just documentation.

Reply:
Ok, that makes sense to me.